lib/tictactoe.rb

```diff
@@ -33,6 +37,28 @@ def begin_player_turn
 
   attr_accessor :board, :current_token, :io, :presenter
 
+  def winning_combinations
+    [
+      [0, 1, 2],
+      [3, 4, 5],
+      [6, 7, 8],
+
+      [0, 3, 6],
+      [1, 4, 7],
+      [2, 5, 8],
+
+      [0, 4, 7],
+      [2, 4, 6]
+    ]
```

Comment:
This logic is specific to a 3x3 board, but since `Board` isn't limited to 3x3, `TicTacToe` could conceivably be interacting with a `Board` with different dimensions. That seems like a contract violation.